### PR TITLE
Fix builds broken by libportal update

### PR DESCRIPTION
**Pull request by jtojnar**
- [x] gcolor3
- [x] gnome-builder
- [x] gnome-todo
- [x] epiphany
- [x] nautilus
- [x] eog
- [x] xdg-desktop-portal

### CODE REVIEW

File: pkgs/desktops/gnome/core/nautilus/default.nix [hunk 1]
```diff
@@ -52,6 +53,17 @@ stdenv.mkDerivation rec {
...
     })
+
+    # Fix build with latest libportal
+    # https://gitlab.gnome.org/GNOME/nautilus/-/merge_requests/749
+    (fetchpatch {
+      url = "https://gitlab.gnome.org/GNOME/nautilus/-/commit/55cfd66ccca391fc144f5863ff6bfc1f3b137e2d.patch";
+      sha256 = "xSb9l7xxEYpAwmdmeWT/t7Z9Ck3DPtsODzbReQW/Q70=";
+      excludes = [
+        "build-aux/flatpak/org.gnome.Nautilus.json"
+        "build-aux/flatpak/org.gnome.Nautilus.yml"
```
Comment (dasj19) [inline pkgs/desktops/gnome/core/nautilus/default.nix:64]: why exculding these two files?

**Review by dasj19 — COMMENTED** [review]
Looks good, except I don't understand why the nautilus patch has some files excluded.

And most of the patches were accepted upstream. That's always a good sign.

File: pkgs/desktops/gnome/core/nautilus/default.nix (hunk 1 above)
Comment (jtojnar) [reply pkgs/desktops/gnome/core/nautilus/default.nix:64]: Due to merge conflicts and irrelevancy.

Comment (dasj19) [reply pkgs/desktops/gnome/core/nautilus/default.nix:64]: Ok I can see now that building flatpaks is irrelevant for us. Thanks for explaining.

**Review by dasj19 — APPROVED** [review]

**Pull request merged**